### DISCUSSION TITLE
How to drag and drop a link in joint js plus

### DISCUSSION THREAD

**Discussion by ghost** [clientIO/joint#1919] [Q&A]
Hi team

I created a link using this  
![diagram](https://user-images.githubusercontent.com/117299732/204752476-9d054860-afe3-47af-bf73-64f49e08ead0.png)
{
          type: "standard.Link",
          size: { width: 60, height: 60 },
          source: { x: 20, y: 20 },
          target: { x: 130, y: 20 },
        attrs: {
            line: {
              stroke: "#222138",
              sourceMarker: {
                fill: "#31d0c6",
                stroke: "none",
                d: "M 5 -10 L -15 0 L 5 10 Z",
              },
              targetMarker: {
                fill: "#fe854f",
                stroke: "none",
                d: "M 5 -10 L -15 0 L 5 10 Z",
              },
            },
          },
        }

In the stencil paper.
And I am trying to drag and drop the link to the normal paper. that time it's not working, it's showing an error like this. How can I solve this issue?

In the stencil paper.
![error](https://user-images.githubusercontent.com/117299732/204752809-00cab3c6-5359-402b-8f3e-84010296b628.png)

Actual need is 

I want to create a diagram like a pipeline structure.  (Crete flexible lines)

![Actual](https://user-images.githubusercontent.com/117299732/204752886-87dffef2-0b51-41a3-bd96-2ed437742a60.png)

**Comment by kumilingus** [comment]
Duplicate of https://github.com/clientIO/joint/discussions/1659.

**Reply by ghost** [reply]
Thanks, @kumilingus
